osidb/filters.py

```diff
@@ -258,6 +259,19 @@ class Meta:
             + DATE_LOOKUP_EXPRS,
             "cvss_scores__uuid": ["exact"],
             "cvss_scores__vector": ["exact"],
+            # Reference fields
+            "references__created_dt": ["exact"]
+            + LT_GT_LOOKUP_EXPRS
+            + LTE_GTE_LOOKUP_EXPRS
+            + DATE_LOOKUP_EXPRS,
+            "references__description": ["exact"],
+            "references__type": ["exact"],
+            "references__updated_dt": ["exact"]
+            + LT_GT_LOOKUP_EXPRS
+            + LTE_GTE_LOOKUP_EXPRS
+            + DATE_LOOKUP_EXPRS,
+            "references__url": ["exact"],
+            "references__uuid": ["exact"],
```

Comment:
Not really a remark for *you* specifically nor a blocker for merging this PR, but there is a clear pattern here for `datetime` fields, we could create a helper function for those and spare a lot of boilerplate code.